doc/Core.xml

```diff
@@ -6131,7 +6131,38 @@ onvif://www.onvif.org/name/ARV-453
     </section>
     <section xml:id="_Toc260402083">
       <title>Storage Configuration</title>
-      <para>The following operations allow client to configure the storage configuration data of device. The storage configuration can refer to DAS, NAS, and CDMI Server.</para>
+      <para>
+        The following operations allows a client to configure the storage configuration data of a device.
+      </para>
+      <section>
+        <title>Authentication</title>
+        <para>
+          Depending on the <literal>type</literal> of <literal>StorageConfigurationData</literal>, the device shall interpret fields in a special way:
+        </para>
+        <para>
+          <variablelist>
+            <varlistentry>
+              <term>ObjectStorageS3</term>
+              <listitem>
+                <para>
+                  The device shall use the <literal>StorageUri</literal> as the base endpoint of the object storage.
+                  This endpoint shall includes the bucket name, either as a subdomain or in the path of the URI, depending on the format supported by the object storage.
+                </para>
+                <para>
+                  The device shall use the <literal>UserName</literal> as the access key ID of the object storage.
+                </para>
+                <para>
+                  The device shall use the <literal>Password</literal> as the secret access key of the object storage.
+                </para>
+                <para>
+                  The device shall use the <literal>Region</literal> as the region of the object storage.
+                  If no <literal>Region</literal> is configured, the device shall use <literal>us-east-1</literal> as the region.
+                </para>
+              </listitem>
+            </varlistentry>
+          </variablelist>
+        </para>
+      </section>
```

Comment:
Add ObjectStoreAzure:

- StorageUri: Service URL including container name
- UserName: Account name
- Password: Password

Reply:
I've looked at the Azure API and I was thinking more of providing a SAS-token to devices to authenticate to the storage instead of trying to manage account credentials per device.
https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview#sas-token

Reply:
So for the storage URI, this will be something like `https://<account>.blob.core.windows.net/<container>`?

Reply:
In all cases, the client has to provide the way to authenticate. Wouldn't providing the SAS as the password be sufficient?

Reply:
> So for the storage URI, this will be something like `https://<account>.blob.core.windows.net/<container>`?

Right, so forget about Account name in UserName.

Reply:
> In all cases, the client has to provide the way to authenticate. Wouldn't providing the SAS as the password be sufficient?

Can it be used as password? I thought this kind of tokens expires after a relatively short period,

Reply:
As discussed, we will be using SAS for now.